ui/raidboss/data/00-misc/test.ts

```diff
@@ -279,24 +282,24 @@ const triggerSet: TriggerSet<Data> = {
         // 50 seconds
         maxDuration: 50000,
       },
-      (ret) => {
-        const me = ret.combatants.find((c) => c.Name === data.me);
-        const dummyName = strikingDummyNames[data.lang] ?? strikingDummyNames['en'];
-        const dummies = ret.combatants.filter((c) => c.Name === dummyName);
-        if (me && dummies) {
-          for (const dummy of dummies) {
-            const distX = Math.abs(me.PosX - dummy.PosX);
-            const distY = Math.abs(me.PosY - dummy.PosY);
-            const dist = Math.hypot(distX, distY);
-            console.log(`test watch: distX = ${distX}; distY = ${distY}; dist = ${dist}`);
-            if (dist < 5)
-              return true;
+        (ret) => {
```

Comment:
:+1: (Although maybe `() => value` for a multi-line call should just be rewritten?)